pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -69,8 +69,9 @@ func parseConfig(config *zconfig.EdgeDevConfig, getconfigCtx *getconfigContext,
 		parseSystemAdapterConfig(config, getconfigCtx, forceSystemAdaptersParse)
 		parseBaseOsConfig(getconfigCtx, config)
 		parseNetworkInstanceConfig(config, getconfigCtx)
-		parseAppInstanceConfig(config, getconfigCtx)
+		//parseAppInstanceConfig(config, getconfigCtx)
```

Comment:
How do app instances run with this commented out?

Reply:
Missed it out. Will uncomment it. Thanks!

Reply:
Resolved.